python/lsst/meas/algorithms/gaussianPsfFactory.py

```diff
@@ -53,8 +53,8 @@ class GaussianPsfFactory(Config):
         check=isPositive,
     )
     sizeFactor = Field(
-        doc="Kernel size as a factor of fwhm (dimensionless); "
-        + "size = sizeFactor * fwhm; ignored if size is not None",
+        doc = "Kernel size as a factor of fwhm (dimensionless); " +
```

Comment:
I didn't think the spaces should be there for keyword args?

Reply:
I checked, and @r-owen is correct according to the style guide.  This makes me sad; I don't think whitespace around `=` should be affected at all by how we split lines.  But I also don't have the energy to RFC a change.